packages/core/src/components/DateField/DateField.jsx

```diff
@@ -106,6 +106,9 @@ export class DateField extends React.PureComponent {
             value={this.props.monthValue}
             aria-describedby={labelId}
           />
+          <span role="presentation" className="ds-c-datefield__separator">
```

Comment:
Nice 👌 `role="presentation"`

Reply:
Please add `aria-hidden="true"` to these `<span>` tags also. I've been testing with the big three screen readers and their preferred browsers. The "slash" being read out feels not quite right between the text labels.

Reply:
@1Copenut Updated, thanks for catching this.

I agree that the slash should not be read out. I was hoping `role="presentation"` would do the trick.